ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,3 @@
 <launch>
-  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen"/>
+  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen" ns="create1"/>
```

Comment:
Rebase against `kinetic-devel` to have a fix for this. Also, don't push this kind of files that are not related to the PR.